changelogs/unreleased/gh-6092-invalid-listen-address-make-tarantool-blind.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/core
+
+* Fixed the fact that using of invalid listen address made
+  tarantool blind. Now in case of invalid listen address
+  tarantool still listen old one (gh-6092).
```

Comment:
Let's rephrase:

Fixed box.cfg.listen not reverted to the old address in case the new one is invalid (gh-6092).

Reply:
fixed